ionic_user_interface/src/views/RequestGym.vue

```diff
@@ -31,34 +31,32 @@ export default defineComponent({
 });
 </script>
 
-<style scoped>
-#container {
+<style scoped lang="scss">
+.my-container {
   text-align: center;
-  position: absolute;
-  left: 0;
-  right: 0;
-}
-
-#container strong {
-  font-size: 3em;
-  line-height: 2em;
-}
-
-#container p {
-  font-size: 1.6em;
-  line-height: 1em;
-  color: #8c8c8c;
-  margin: 0;
-}
-
-#container a {
-  text-decoration: none;
-}
-
-#container .description {
-  font-size: 1.1em;
-  line-height: 1.4em;
-  color: var(--ion-color-medium-shade);
-  margin: 1rem;
+  margin: 0 auto;
+
+  strong {
+    font-size: 3em;
+    line-height: 2em;
+  }
+
+  p {
+    font-size: 1.6em;
+    line-height: 1em;
+    color: #8c8c8c;
```

Comment:
Should we change all the subtitles to an ion color instead of hardcoding? I think a good one will be `color: var(--ion-color-medium-shade);`

Reply:
ok, will search for hard-coded colors and change them